tests/test_client.py

```diff
@@ -47,17 +45,32 @@ def test_client_enqueue_and_get_result(dbos: DBOS, client: DBOSClient) -> None:
     result = handle.get_result()
     assert result == '42-test-{"first": "John", "last": "Doe", "age": 30}'
 
+    list_results = client.list_workflows()
+    assert len(list_results) == 1
+    assert list_results[0].workflow_id == wfid
+    assert list_results[0].status == "SUCCESS"
+
 
 def test_enqueue_with_timeout(dbos: DBOS, client: DBOSClient) -> None:
     run_client_collateral()
 
+    wfid = str(uuid.uuid4())
     options: EnqueueOptions = {
         "queue_name": "test_queue",
         "workflow_name": "blocked_workflow",
-        "workflow_timeout": 0.1,
+        "workflow_timeout": 5,
+        "workflow_id": wfid,
     }
 
     handle: WorkflowHandle[str] = client.enqueue(options)
+
+    time.sleep(1)
```

Comment:
Why changing the test here? Also we should avoid using `time.sleep()`

Reply:
fixed thanks